person/models.py

```diff
@@ -1,14 +1,33 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 from django.utils import timezone
+from django.core.validators import MinValueValidator
+from django.core.validators import RegexValidator
 
 SEX_TYPES = (
     ('F','Femenino'),
     ('M', 'Masculino'),
     ('O', 'Otro'),
 )
 
-class Person(AbstractUser):
+PAYMENT_METHOD = (
```

Comment:
Maybe it would be a good idea to include a new "Efectivo" status? To take into account those people who find it easier to use this method.

Reply:
ok